test/test_beanstalk_utils.py

```diff
@@ -114,38 +122,140 @@ def communicate(self):
                     'AWS_FAKE_SECRET': 'amazon'
                 }
 
-def test_magic_cnames():
+
+def _ip_addresses(hostname):
+    return sorted(socket.gethostbyname_ex(hostname)[2])
+
+
+def test_magic_cnames_by_production_ip_address():
```

Comment:
I get that this should always work, but does it really belong in a unit test here?

Reply:
If's a form of consistency checking. Whether we should reach out to the net to test it is another question but otherwise it's just a magic constant.

If we had a rule against there being unit tests that talked to the net, I'd move it to integration tests but it seems to me OK here for now.